src/components/NavBar/NavIcon.css.ts

```diff
@@ -15,6 +15,7 @@ export const navIcon = style([
     cursor: 'pointer',
     borderRadius: '8',
     transition: '250',
+    marginRight: '4',
```

Comment:
This changes the style of the generic `NavIcon` component which is also shared with the NFT Shopping bag icon as well as the overflow menu. Updating the NFT Shopping bag is unintended. But you can move this style change one level up to the flex `Box` that wraps this `NavIcon` in `MenuDropdown.tsx`

Reply:
That should work , will try it.

Reply:
Added the margin to the box.